tools/model_tools/src/openvino/model_zoo/quantize_engine/quantizer.py

```diff
@@ -92,20 +92,28 @@ def get_pot_cmd_prefix(self) -> List[str]:
         return pot_cmd_prefix
 
 
-    def quantize(self, reporter, model, precision, target_device, pot_cmd_prefix, pot_env) -> bool:
+    def quantize(self, reporter, model, precision, target_device, pot_cmd_prefix, pot_env, model_root=None) -> bool:
         input_precision = _common.KNOWN_QUANTIZED_PRECISIONS[precision]
 
-        pot_config_base_path = _common.MODEL_ROOT / model.subdirectory / 'quantization.yml'
+        model_root = _common.MODEL_ROOT if model_root is None else model_root
+        pot_config_base_path = model_root / model.subdirectory / 'quantization.yml'
 
```

Comment:
Please move
```
reporter.print_section_heading('{}Quantizing {} from {} to {}',
            '(DRY RUN) ' if self.dry_run else '', model.name, input_precision, precision)
```
here to have consistent output structure.

Reply:
The idea was to move printing of section heading above all checks. So that we have all the messages about `quantization.yml` or `accuracy-check.yml` after the header. For example:
```
========== Quantizing se-inception from FP16 to FP16-INT8
Unable to locate quantization.yml in /home/jkamelin/github/fork/one-zoo/open_model_zoo/models/public/se-inception/quantization.yml, loading default POT config
```